java-sdk-logging/license-checks.xml

```diff
@@ -5,6 +5,6 @@
 <module name="Checker">
   <module name="RegexpHeader">
     <property name="fileExtensions" value="java"/>
-    <property name="headerFile" value="./java-sdk-logging/java.header"/>
+    <property name="headerFile" value="./java.header"/>
```

Comment:
Probably a separate topic: why do we have a `java.header` file in each submodule? I think we can have one under root, and all submodules reference it?

Reply:
Maybe a follow up PR to create a `java.header` in root.